### DISCUSSION TITLE
Recognize Drum scores (at least cymbal ones)

### DISCUSSION THREAD

**Discussion by Domigome** [BreezeWhite/oemer#10] [Ideas]
How difficult would this be?

I think including the "x" head would allow for a quite ok drum score recognition to start with. 

At least it would give us a very nice starting point for drummers! :)

**Comment by BreezeWhite** [comment]
It's totally different from the current algorithm. You need to re-train the deep learning model. To train the model, you thus need a large amount of data, and as far as I know, there is no such dataset designed for recognizing percussion symbols.

**Reply by Domigome** [reply]
The algorithm detect the note head in a different pass than stems and flags, right?

I was wondering if in that case a simple OCR could detect if the headnote is a "x".
- If it is good, we got a cymbal for drums.
- If not, then we can go on and detect with the existing model.

Would something like this be more realistic?
